gtest_parallel.py

```diff
@@ -185,6 +184,11 @@ def _normalize(string):
 
   @staticmethod
   def _logname(output_dir, test_binary, test_name, execution_number):
+    if output_dir is None:
```

Comment:
Comment here saying `# Store logs to temporary files while running if there's no output_dir.` or something like that.